### ISSUE TITLE
Installation issue unable to find .git\hooks folder 

### ISSUE THREAD

**Issue by brendonco** [typicode/husky#615]
"husky": "^3.1.0",

ENOENT: no such file or directory, mkdir 'node_modules\husky\.git\hooks'
husky > Failed to install

"husky": {
    "hooks": {
      "pre-commit": "lint-staged && npm test"
    }
  },
  "lint-staged": {
    "*.js": [
      "eslint",
      "git add"
    ]
  },

on pre-commit, doesnt run the lint staged and npm test

**Comment by nlapshin** [comment]
+1

**Comment by TravisYeah** [comment]
I fixed this by closing my editor before running the install. The editor was blocking the files from being created.

**Comment by tylergaw** [comment]
I ran into this same issue today. My problem was that I wasn't on a new-enough version of git. From the [install docs](https://github.com/typicode/husky#install):

> Requires Node >= 8.6.0 and Git >= 2.13.0

I was on Git `2.5.x`. I updated to the latest git `2.24.1` via Homebrew then:

```
npm uninstall husky --save-dev
npm install husky --save-dev
```

With that, the install and hooks setup went as expected. A pre-commit hook I set up also ran as expected.
